packages/@orbit/jsonapi/src/jsonapi-serializer.ts

```diff
@@ -309,6 +325,104 @@ export class JSONAPISerializer
     return result;
   }
 
+  deserializeOperationsDocument(
+    document: ResourceOperationsDocument
+  ): RecordOperation[] {
+    return this.deserializeOperations(document.operations);
+  }
+
+  deserializeOperations(operations: ResourceOperation[]): RecordOperation[] {
+    return operations.map(operation => {
```

Comment:
I'd prefer to have another level, `deserializeOperation`, that either performs all the individual operation deserialization or invokes methods such as `deserialize[X]Operation`.